pkg/runner/collector_selector.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package runner
+
+import (
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/redhat-partner-solutions/vse-sync-testsuite/pkg/collectors"
+)
+
+var (
+	optionalCollectorNames []string
```

Comment:
Is there a particular reason for populating these vars in the `init`, rather than at declaration? I think this would work fine:
```suggestion
	optionalCollectorNames = []string{collectors.DPLLCollectorName, collectors.GPSCollectorName}
```

Same for the other list (and possibly the flag default value)